### PR TITLE
Change phrasing for unseal key notification

### PR DESCRIPTION
**Pull request by cbednarski**

### CODE REVIEW

File: command/init.go [hunk 1]
```diff
@@ -47,17 +47,14 @@ func (c *InitCommand) Run(args []string) int {
...
-			"Vault initialized with %d keys and a key threshold of %d!\n\n"+
-			"Please securely distribute the above keys. Whenever a Vault server\n"+
-			"is started, it must be unsealed with %d (the threshold) of the\n"+
-			"keys above (any of the keys, as long as the total number equals\n"+
-			"the threshold).\n\n"+
-			"Vault does not store the original master key. If you lose the keys\n"+
-			"above such that you no longer have the minimum number (the\n"+
-			"threshold), then your Vault will not be able to be unsealed.",
+			"Vault initialized with %d keys and a key threshold of %d. Please\n"+
+			"securely distribute the above keys. When Vault restarts you must\n"+
```
Comment (sethvargo) [inline command/init.go]: I think we should say "if vault is restarted" or something like:

> If the Vault is re-sealed or stopped, you must provide the keys to unseal it again.

**Comment by cbednarski** [comment]
@armon Feel free to discard this if you feel it changes the intent.

File: command/init.go (hunk 1 above)
Comment (cbednarski) [inline command/init.go]: @sethvargo Thanks, good clarifications.

**Comment by sethvargo** [comment]
:+1:

**Pull request merged**